Documentation/compatibility/interop-event-handlers-safearray.md

```diff
@@ -0,0 +1,29 @@
+## .NET COM successfully marshals ByRef SafeArray parameters on events
+
+### Scope
+Minor
+
+### Version Introduced
+4.8
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Prior to this change an ByRef SafeArray parameter on a COM event would fail to marshal back to native code.  With this change the SafeArray is now marshalled successfully.
```

Comment:
```suggestion
In the .NET Framework 4.7.2 and earlier versions, a ByRef SafeArray parameter on a COM event would fail to marshal back to native code.  With this change the SafeArray is now marshalled successfully.
```
Perhaps make SafeArray a link: /windows/desktop/api/oaidl/ns-oaidl-safearray

Reply:
Will do